test/e2e/manifest.yaml

```diff
@@ -0,0 +1,27 @@
+StorageClass:
+  FromExistingClassName: standard-csi
+SnapshotClass:
+  FromName: true
+DriverInfo:
+  Name: pd.csi.storage.gke.io
+  SupportedFsType:
+    xfs: {}
+    ext4: {}
+  SupportedMountOption:
+    debug:
+    nouid32:
+  SupportedSizeRange:
+    Min: 1Gi
+    Max: 64Ti
+  TopologyKeys:
+    - topology.gke.io/zone
+  Capabilities:
+    persistence: true
+    fsGroup: true
+    block: true
+    exec: true
+    volumeLimits: false
```

Comment:
Does the driver support limits? Please add a comment if so and we want to avoid creating ~100s of PVs.

Reply:
Added comment explaining why this cap is disabled